documentation/release-latest/docs/install/integrations.md

```diff
@@ -216,6 +216,14 @@ See [whirm/flycheck-kotlin](https://github.com/whirm/flycheck-kotlin).
 
 See [w0rp/ale](https://github.com/w0rp/ale).
 
+## [TCA](http://tca.tencent.com/) integration
+
+[Tencent Cloud Code Analysis](http://tca.tencent.com/) (TCA for short, code-named CodeDog inside the company early) is a comprehensive platform for code analysis and issue tracking. TCA consist of three components, server, web and client. It integrates of a number of self-developed tools, and also supports dynamic integration of code analysis tools in various programming languages.
```

Comment:
Please, move your changes below the reference to Mega-linter.

The changes also have to be applied to the snapshot documentation. Otherwise, your change will be lost at the next release.

Reply:
DONE